thorn/src/kernel/kernel.c

```diff
@@ -93,25 +97,27 @@ void kernel_main (int processor_id) {
     printf ("Hello, from processor %d\n\r", processor_id);
 
     current_processor++;
-
-    if (processor_id == 0) {
-        while (current_processor != 3)
-            ;
-
-        draw ();
-
-        //        int res = copy_process (PF_KTHREAD, (unsigned long) & kernel_process, 0, 0);
-        //        if (res < 0) {
-        //            printf ("error while starting kernel process");
-        //            return;
-        //        }
-        //
-        //        while (1) {
-        //            schedule ();
-        //        }
-        LOG ("DONE PRINTING");
+    while (current_processor != 3)
```

Comment:
also what's the issue with processor 3?

Reply:
I don't know. Maybe we accidentally awaken processor 3 prematurely and it crashes

Reply:
But what about the other 2 non-main processors? At least right now this doesn't do anything anyways

Reply:
me neither. I'm looking into it right now